tests/sentry/feedback/test_project_feedback_details.py

```diff
@@ -221,3 +221,110 @@ def test_other_project(self):
             )
             response = self.client.get(path)
             assert response.status_code == 404
+
+    def test_successful_delete(self):
+        Feedback.objects.create(
```

Comment:
at this point might be helpful to create a mock function similar to https://github.com/getsentry/sentry/blob/a97d7d66b4fabffdb2b7ddbed31deaed48b4301e/src/sentry/replays/testutils.py#L124. could create a `testutils` file inside of feedback.